docs/reference/commandline/node_promote.md

```diff
@@ -26,8 +26,8 @@ Options:
 
 ## Description
 
-Promotes a node to manager. This command targets a docker engine that is a
-manager in the swarm.
+Promotes a node to manager. This command targets a docker engine that is not a manager 
```

Comment:
Hm... I think the confusing here is that;

- This command must be run against an engine that _is_ a manager (only managers can promote/demote a node)
- But the _node_ to promote is not yet a manager

Perhaps something like (better suggestions welcome 😅);

```Markdown
This command can only be executed on a manager node.
```